src/core/port.ts

```diff
@@ -34,7 +34,7 @@ export abstract class Port<T extends Present> extends Trigger {
     public isPresent() {
 
         Log.debug(this, () => "In isPresent()...")
-        Log.debug(this, () => "value: " + this.value);
+        Log.debug(this, () => "value: " + this.value?.toString());
```

Comment:
Why isn't this part like `value.toString()` ?

Reply:
[The question mark handles the fact that `this.value` is nullable](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Reply:
Ah thank you. :)